src/cli/new_mnemonic.rs

```diff
@@ -78,9 +85,26 @@ pub fn run<'a>(sub_match: &ArgMatches<'a>) {
         .parse::<u32>()
         .expect("invalid number of validators");
 
-    let chain = sub_match
-        .value_of("chain")
-        .expect("missing chain identifier");
+    let chain = sub_match.value_of("chain");
+
+    let chain_spec_file = match chain {
+        Some(_) => None,
+        None => Some(
+            sub_match
+                .value_of("testnet_config")
+                .expect("must pass testnet config path if not using chain")
+                .to_string(),
+        ),
+    };
+
+    let chain = if chain_spec_file.is_some() && chain.is_some() {
```

Comment:
I think it would be better to force user to pass chain and check verify that `minimal` and `testnet` pass a spec also

Reply:
In case if people need to use custom config, `--testnet_config` replaces `--chain`,  it's less typing for users so I'd prefer not to make `--chain` required.

Reply:
maybe `custom` makes more sense actually, it makes it more explicit. i don't think it's too much to type, but also keeps the overall usage of the cli similar